src/main/java/org/electronicReferences/Main.java

```diff
@@ -0,0 +1,6 @@
+package org.electronicReferences;
+public class Main {
+    public static void main(String[] args) {
+       //
```

Comment:
Why is this main method empty? If I understand it correctly, it should be an entrypoint for launching `SpringBootApplication`

Reply:
Done